wfe/wfe_test.go

```diff
@@ -190,7 +190,7 @@ func (ra *MockRegistrationAuthority) NewCertificate(ctx context.Context, req cor
 }
 
 func (ra *MockRegistrationAuthority) UpdateRegistration(ctx context.Context, reg core.Registration, updated core.Registration) (core.Registration, error) {
-	if reg.Key != updated.Key {
+	if reg.Key.Key != updated.Key.Key {
```

Comment:
I got curious, and found this Stack Overflow answer that explains equality testing of `interface{}`: http://stackoverflow.com/a/34246225/363869. But still probably better to do `publicKeysEqual`.